tldw_Server_API/app/api/v1/endpoints/mlx.py

```
@@ -0,0 +1,71 @@
+
+router = APIRouter()
+
+
+@router.post(
+    "/llm/providers/mlx/load",
+    summary="Load or swap the active MLX model",
+    dependencies=[Depends(check_rate_limit)],
+)
+async def load_mlx_model(
+    payload: MLXLoadRequest,
+    admin=Depends(require_admin),
```

Ruff (0.14.6)

Unused function argument: `admin`

(ARG001)